sponsors/index.html

```diff
@@ -25,7 +25,7 @@ <h1>Sponsoren</h1>
 
 <section>
     <div class="container">
-        <h2>Unterstützer</h2>
+        <h2>Supporters</h2>

```

Comment:
@PatrickGoettsch is there any specific wording in our sponsoring contracts for the wording of the sponsor classes, or can we edit this freely?

Reply:
Patrick said that this is fine (via telegram)